common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4361,20 +4385,93 @@ object CelebornConf extends Logging {
   val DYNAMIC_CONFIG_STORE_BACKEND: ConfigEntry[String] =
     buildConf("celeborn.dynamicConfig.store.backend")
       .categories("master", "worker")
-      .doc("Store backend for dynamic config. Available options: NONE, FS. Note: NONE means disabling dynamic config store.")
+      .doc("Store backend for dynamic config service. Available options: FS, DB.")
       .version("0.4.0")
       .stringConf
       .transform(_.toUpperCase(Locale.ROOT))
-      .checkValues(Set("NONE", "FS"))
-      .createWithDefault("NONE")
+      .checkValues(Set("FS", "DB"))
+      .createWithDefault("FS")
 
-  val DYNAMIC_CONFIG_REFRESH_INTERVAL: ConfigEntry[Long] =
+  val DYNAMIC_CONFIG_REFRESH_INTERVAL: OptionalConfigEntry[Long] =
     buildConf("celeborn.dynamicConfig.refresh.interval")
```

Comment:
Does the dynamic config work well when `celeborn.dynamicConfig.refresh.interval` is not configured?

Reply:
Make sense, I add a new configuration `celeborn.dynamicConfig.refresh.enabled` to switch, that's more clear now.